apps/blockchain/lib/blockchain/account.ex

```diff
@@ -514,7 +514,7 @@ defmodule Blockchain.Account do
       iex> Blockchain.Account.get_storage(updated_state, <<01::160>>, 5)
       {:ok, 9}
   """
-  @spec put_storage(EVM.state(), EVM.address(), integer(), integer()) :: EVM.state | { EVM.state, t, t }
+  @spec put_storage(EVM.state(), EVM.address(), integer(), integer()) :: EVM.state
```

Comment:
I'm sorry but tuple is actually needed
```
if return_accounts do
      {updated_state, account, updated_account}
    else
      updated_state
end
```

Reply:
return_accounts will always be false in put_storage case won't it? So it won't return a tuple.

Reply:
yes, you're right